### DISCUSSION TITLE
OPENGL Shaders

### DISCUSSION THREAD

**Discussion by jvernet** [86Box/86Box#1392] [Q&A] [answered]
Hello,

Nice to see OPENGL shaders coming in 86Box. However, is there a way to use multiple shaders ? And how can we set shaders partameters ?
Im'h using Libretro shaders, crt/easymode_halation is the best to emulate CRT (glow, size, forms, scanline, mask, even emulate jittering).

**Comment by ts-korhonen** [comment, marked as answer]
Hi,
The OpenGL Core renderer is a work-in-progress. The version in experimental build only works with single-pass shaders that don't use previous frames or scaling. The plan is to support most of libretro shaders eventually.

Currently shader parameters can only by set in the shader file itself.
